src/sections/Community/Handbook/Handbook.style.js

```diff
@@ -135,17 +134,18 @@ export const HandbookWrapper = styled.div`
     }
 
     .community-home-container{
-        padding: 1rem 0;
-        padding-bottom: 4rem;
-        display: flex;
-        flex-wrap : wrap;
-        align-items : center;
-        justify-content: center;
+      padding: 1rem 0;
+      padding-bottom: 4rem;
+      display: flex;
+      flex-wrap : wrap;
+      align-items : center;
+      justify-content: center;
```

Comment:
and this, I think if you revert this whole file with these "formatting changes",  it would be fine as it is not needed in my opinion :)